src/role-helpers.js

```diff
@@ -138,23 +139,30 @@ function getRoles(container, {hidden = false} = {}) {
     }, {})
 }
 
-function prettyRoles(dom, {hidden}) {
+function prettyRoles(dom, {hidden, includeName}) {
   const roles = getRoles(dom, {hidden})
 
   return Object.entries(roles)
     .map(([role, elements]) => {
       const delimiterBar = '-'.repeat(50)
       const elementsString = elements
-        .map(el => prettyDOM(el.cloneNode(false)))
+        .map(
+          el =>
+            `${
+              includeName === true
+                ? `Name "${computeAccessibleName(el)}":\n`
+                : ''
+            }${prettyDOM(el.cloneNode(false))}`,
+        )
         .join('\n\n')
 
       return `${role}:\n\n${elementsString}\n\n${delimiterBar}`
     })
     .join('\n')
 }
 
-const logRoles = (dom, {hidden = false} = {}) =>
-  console.log(prettyRoles(dom, {hidden}))
+const logRoles = (dom, {hidden = false, includeName = false} = {}) =>
```

Comment:
Why would we not want to always include the name?

Reply:
Haven't thought much about this. Just wanted to not change existing behavior. 
Since this is only used by library consumers it might be a good idea to include the name by default to promote that we now support accessible name queries?

Reply:
Yes, since it's debug output, it seems fine to change.